examples/gemma/layers.py

```diff
@@ -51,7 +52,7 @@ def __init__(self, dim: int, *, rngs: nnx.Rngs):
 
   def __call__(self, x: Array) -> Array:
     var = jnp.mean(jnp.square(x), axis=-1, keepdims=True)
-    normed_inputs = jnp.asarray(x * jnp.reciprocal(jnp.sqrt(var + 1e-06)))
+    normed_inputs = jnp.asarray(x * jax.lax.rsqrt(var + 1e-06))
```

Comment:
This what also MaxText is doing: https://github.com/AI-Hypercomputer/maxtext/blob/33339821e96327d25613ace9d28c7a8789698586/MaxText/layers/normalizations.py#L60